snmp/datadog_checks/snmp/snmp.py

```diff
@@ -388,6 +389,9 @@ def _check_with_config(self, config):
                 error = 'Failed to collect metrics for {} - {}'.format(self._get_instance_name(instance), e)
             self.warning(error)
         finally:
+            # At this point, `tags` might includes some extra tags added in try clause
+            self.gauge('snmp.devices_monitored', 1, tags=tags)
```

Comment:
Can we expand the comment to explain why this gauge is submitted with value `1`? I'm assuming there's something to do with what is in `tags` and how it would work in the UI, but I'm not sure.

Reply:
Expanded comment.